db/version_set.h

```diff
@@ -1672,13 +1675,15 @@ class ReactiveVersionSet : public VersionSet {
 
  private:
   std::unique_ptr<ManifestTailer> manifest_tailer_;
-
+  // TODO: plumb Env::IOActivity
+  const ReadOptions read_options_;
```

Comment:
[Note] Unfortunately manifest_tailer_ breaks my assumption of the lifetime of version edit handler is same as the function where the handler is. Instead this one is same as ReactiveVersionSet .

So I am still debating whether to pass ReadOptions through most version edit handler's functions (e.g, Iterate(), ...) or constructing the hanlder with ReadOptions of function scope, which leading to weird stuff here in ReactiveVersionSet